recipes/tbb/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.7
```

Comment:
Though you can get these errors because the version would be unacceptable whether or not you have the SDK. Kind of interesting, but does happen.